packages/transaction-controller/src/utils/layer1-gas-fee-flow.test.ts

```diff
@@ -0,0 +1,43 @@
+import { CHAIN_IDS } from '../constants';
+import type { Layer1GasFeeFlow, TransactionMeta } from '../types';
+import { getLayer1GasFeeFlow } from './layer1-gas-fee-flow';
+
+const MOCK_UNSUPPORTED_CHAIN_ID = '0xunsupported';
+
+describe('getLayer1GasFeeFlow', () => {
+  const mockLayer1GasFeeFlows = [
```

Comment:
Rather than hardcoding the implementations, is it better we have a function to create a mock using jest functions?

As we do elsewhere and in the `gas-flows.test.ts` via `createGasFeeFlowMock`.

Reply:
Done in https://github.com/MetaMask/core/pull/3944/commits/fa567b3f7eebe1b897bd615c912cb32c6de4801f